[2024-09-20T08:20:42.788+0000] {processor.py:186} INFO - Started process (PID=4296) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:20:42.790+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:20:42.794+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:42.794+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:20:42.819+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:20:42.954+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:42.953+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG:new_dag
[2024-09-20T08:20:42.972+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:42.972+0000] {override.py:1900} INFO - Created Permission View: can edit on DAG:new_dag
[2024-09-20T08:20:42.984+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:42.983+0000] {override.py:1900} INFO - Created Permission View: can read on DAG:new_dag
[2024-09-20T08:20:42.997+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:42.997+0000] {override.py:1900} INFO - Created Permission View: can create on DAG Run:new_dag
[2024-09-20T08:20:43.010+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:43.009+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG Run:new_dag
[2024-09-20T08:20:43.021+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:43.021+0000] {override.py:1900} INFO - Created Permission View: menu access on DAG Run:new_dag
[2024-09-20T08:20:43.032+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:43.032+0000] {override.py:1900} INFO - Created Permission View: can read on DAG Run:new_dag
[2024-09-20T08:20:43.034+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:43.033+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:20:43.050+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:43.049+0000] {dag.py:3252} INFO - Creating ORM DAG for new_dag
[2024-09-20T08:20:43.062+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:20:43.062+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:20:43.084+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.303 seconds
[2024-09-20T08:21:13.238+0000] {processor.py:186} INFO - Started process (PID=4357) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:21:13.241+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:21:13.247+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:21:13.246+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:21:13.271+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:21:13.298+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:21:13.298+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:21:13.328+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:21:13.328+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:21:13.390+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.163 seconds
[2024-09-20T08:21:43.724+0000] {processor.py:186} INFO - Started process (PID=4420) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:21:43.726+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:21:43.731+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:21:43.731+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:21:43.775+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:21:43.808+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:21:43.808+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:21:43.855+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:21:43.854+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:21:43.886+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.170 seconds
[2024-09-20T08:22:14.095+0000] {processor.py:186} INFO - Started process (PID=4483) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:22:14.096+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:22:14.100+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:22:14.099+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:22:14.131+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:22:14.157+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:22:14.157+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:22:14.187+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:22:14.187+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:22:14.229+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.143 seconds
[2024-09-20T08:22:44.472+0000] {processor.py:186} INFO - Started process (PID=4545) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:22:44.473+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:22:44.479+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:22:44.478+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:22:44.507+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:22:44.567+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:22:44.567+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:22:44.618+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:22:44.618+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:22:44.654+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.200 seconds
[2024-09-20T08:23:15.183+0000] {processor.py:186} INFO - Started process (PID=4608) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:23:15.186+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:23:15.189+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:23:15.189+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:23:15.221+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:23:15.252+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:23:15.251+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:23:15.282+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:23:15.281+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:23:15.308+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.140 seconds
[2024-09-20T08:23:45.424+0000] {processor.py:186} INFO - Started process (PID=4671) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:23:45.426+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:23:45.429+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:23:45.429+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:23:45.478+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:23:45.510+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:23:45.510+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:23:45.547+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:23:45.547+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:23:45.579+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.167 seconds
[2024-09-20T08:24:15.666+0000] {processor.py:186} INFO - Started process (PID=4734) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:24:15.667+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:24:15.671+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:24:15.671+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:24:15.699+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:24:15.731+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:24:15.731+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:24:15.767+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:24:15.767+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:24:15.797+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.144 seconds
[2024-09-20T08:24:46.085+0000] {processor.py:186} INFO - Started process (PID=4797) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:24:46.090+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:24:46.094+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:24:46.094+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:24:46.115+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:24:46.142+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:24:46.141+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:24:46.173+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:24:46.173+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:24:46.199+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.122 seconds
[2024-09-20T08:25:16.751+0000] {processor.py:186} INFO - Started process (PID=4860) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:25:16.754+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:25:16.758+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:25:16.757+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:25:16.779+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:25:16.807+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:25:16.807+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:25:16.840+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:25:16.839+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:25:16.903+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.161 seconds
[2024-09-20T08:25:47.119+0000] {processor.py:186} INFO - Started process (PID=4923) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:25:47.120+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:25:47.124+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:25:47.123+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:25:47.153+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:25:47.185+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:25:47.185+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:25:47.223+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:25:47.223+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:25:47.252+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.148 seconds
[2024-09-20T08:26:17.353+0000] {processor.py:186} INFO - Started process (PID=4986) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:26:17.354+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:26:17.360+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:26:17.359+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:26:17.390+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:26:17.423+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:26:17.422+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:26:17.455+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:26:17.455+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:26:17.483+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.141 seconds
[2024-09-20T08:26:47.704+0000] {processor.py:186} INFO - Started process (PID=5049) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:26:47.706+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:26:47.713+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:26:47.712+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:26:47.758+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:26:47.788+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:26:47.788+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:26:47.827+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:26:47.826+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:26:47.893+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.209 seconds
[2024-09-20T08:27:18.225+0000] {processor.py:186} INFO - Started process (PID=5118) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:27:18.227+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:27:18.230+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:27:18.230+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:27:18.256+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:27:18.287+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:27:18.287+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:27:18.320+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:27:18.320+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:27:18.379+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.163 seconds
[2024-09-20T08:27:48.773+0000] {processor.py:186} INFO - Started process (PID=5181) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:27:48.774+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:27:48.778+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:27:48.778+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:27:48.807+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:27:48.840+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:27:48.839+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:27:48.874+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:27:48.873+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:27:48.936+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.172 seconds
[2024-09-20T08:28:19.528+0000] {processor.py:186} INFO - Started process (PID=5244) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:28:19.529+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:28:19.533+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:28:19.533+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:28:19.562+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:28:19.585+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:28:19.585+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:28:19.616+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:28:19.615+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:28:19.681+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.165 seconds
[2024-09-20T08:28:49.787+0000] {processor.py:186} INFO - Started process (PID=5307) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:28:49.789+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:28:49.793+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:28:49.793+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:28:49.820+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:28:49.849+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:28:49.848+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:28:49.884+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:28:49.883+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:28:49.929+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.154 seconds
[2024-09-20T08:29:20.431+0000] {processor.py:186} INFO - Started process (PID=5370) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:29:20.433+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:29:20.436+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:29:20.436+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:29:20.463+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:29:20.488+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:29:20.487+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:29:20.519+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:29:20.519+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:29:20.584+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.164 seconds
[2024-09-20T08:29:50.689+0000] {processor.py:186} INFO - Started process (PID=5433) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:29:50.690+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:29:50.693+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:29:50.693+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:29:50.714+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:29:50.742+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:29:50.741+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:29:50.776+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:29:50.775+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:29:50.839+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.162 seconds
[2024-09-20T08:30:21.563+0000] {processor.py:186} INFO - Started process (PID=5496) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:30:21.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:30:21.571+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:30:21.570+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:30:21.606+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:30:21.647+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:30:21.647+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:30:21.689+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:30:21.689+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:30:21.761+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.212 seconds
[2024-09-20T08:30:51.983+0000] {processor.py:186} INFO - Started process (PID=5559) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:30:51.996+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:30:52.006+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:30:52.005+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:30:52.047+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:30:52.093+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:30:52.092+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:30:52.147+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:30:52.147+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:30:52.225+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.261 seconds
[2024-09-20T08:31:22.368+0000] {processor.py:186} INFO - Started process (PID=5622) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:31:22.370+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:31:22.378+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:31:22.377+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:31:22.414+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:31:22.508+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:31:22.508+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:31:22.592+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:31:22.592+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:31:22.698+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.345 seconds
[2024-09-20T08:31:53.106+0000] {processor.py:186} INFO - Started process (PID=5685) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:31:53.108+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:31:53.112+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:31:53.111+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:31:53.142+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:31:53.170+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:31:53.170+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:31:53.202+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:31:53.202+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:31:53.272+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.182 seconds
[2024-09-20T08:32:23.865+0000] {processor.py:186} INFO - Started process (PID=5748) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:32:23.866+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:32:23.871+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:32:23.870+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:32:23.903+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:32:23.943+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:32:23.942+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:32:23.986+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:32:23.985+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:32:24.057+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.201 seconds
[2024-09-20T08:32:54.149+0000] {processor.py:186} INFO - Started process (PID=5811) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:32:54.152+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:32:54.158+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:32:54.158+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:32:54.179+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:32:54.207+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:32:54.207+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:32:54.239+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:32:54.239+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:32:54.300+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.160 seconds
[2024-09-20T08:33:25.073+0000] {processor.py:186} INFO - Started process (PID=5874) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:33:25.075+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:33:25.078+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:33:25.077+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:33:25.104+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:33:25.134+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:33:25.133+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:33:25.164+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:33:25.164+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:33:25.192+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.136 seconds
[2024-09-20T08:33:55.319+0000] {processor.py:186} INFO - Started process (PID=5938) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:33:55.322+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:33:55.330+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:33:55.328+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:33:55.359+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:33:55.385+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:33:55.385+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:33:55.424+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:33:55.424+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:33:55.461+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.152 seconds
[2024-09-20T08:35:20.920+0000] {processor.py:186} INFO - Started process (PID=35) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:35:20.923+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:35:20.934+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:35:20.933+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:35:20.980+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:35:21.071+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:35:21.071+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:35:21.157+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:35:21.156+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:35:21.218+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.325 seconds
[2024-09-20T08:35:51.930+0000] {processor.py:186} INFO - Started process (PID=97) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:35:51.933+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:35:51.939+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:35:51.939+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:35:51.971+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:35:52.000+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:35:52.000+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:35:52.036+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:35:52.036+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:35:52.070+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.150 seconds
[2024-09-20T08:36:22.650+0000] {processor.py:186} INFO - Started process (PID=160) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:36:22.653+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:36:22.658+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:36:22.658+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:36:22.697+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:36:22.729+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:36:22.729+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:36:22.761+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:36:22.760+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:36:22.789+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.152 seconds
[2024-09-20T08:36:53.025+0000] {processor.py:186} INFO - Started process (PID=223) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:36:53.028+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:36:53.031+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:36:53.031+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:36:53.060+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:36:53.094+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:36:53.093+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:36:53.129+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:36:53.129+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:36:53.160+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.145 seconds
[2024-09-20T08:37:23.405+0000] {processor.py:186} INFO - Started process (PID=286) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T08:37:23.406+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T08:37:23.415+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:37:23.414+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T08:37:23.450+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T08:37:23.480+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:37:23.480+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T08:37:23.520+0000] {logging_mixin.py:190} INFO - [2024-09-20T08:37:23.519+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T08:37:23.557+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.167 seconds
[2024-09-20T10:48:16.250+0000] {processor.py:186} INFO - Started process (PID=40) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:48:16.253+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:48:16.259+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:48:16.258+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:48:16.318+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:48:16.399+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:48:16.398+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T10:48:16.453+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:48:16.452+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T10:48:16.488+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.252 seconds
[2024-09-20T10:48:46.633+0000] {processor.py:186} INFO - Started process (PID=47) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:48:46.635+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:48:46.640+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:48:46.640+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:48:46.684+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:48:46.723+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:48:46.723+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T10:48:46.778+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:48:46.777+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T10:48:46.808+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.185 seconds
[2024-09-20T10:49:17.175+0000] {processor.py:186} INFO - Started process (PID=54) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:49:17.176+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:49:17.180+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:49:17.179+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:49:17.228+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:49:17.270+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:49:17.269+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T10:49:17.301+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:49:17.301+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T10:49:17.332+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.165 seconds
[2024-09-20T10:49:47.542+0000] {processor.py:186} INFO - Started process (PID=61) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:49:47.543+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:49:47.547+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:49:47.547+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:49:47.593+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:49:47.630+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:49:47.629+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T10:49:47.660+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:49:47.660+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T10:49:47.687+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.153 seconds
[2024-09-20T10:50:18.044+0000] {processor.py:186} INFO - Started process (PID=68) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:50:18.045+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:50:18.050+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:50:18.049+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:50:18.093+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:50:18.140+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:50:18.139+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T10:50:18.172+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:50:18.172+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T10:50:18.198+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.164 seconds
[2024-09-20T10:50:48.471+0000] {processor.py:186} INFO - Started process (PID=75) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:50:48.473+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:50:48.478+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:50:48.478+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:50:48.536+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:50:48.585+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:50:48.584+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T10:50:48.621+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:50:48.621+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T10:50:48.652+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.202 seconds
[2024-09-20T10:51:18.924+0000] {processor.py:186} INFO - Started process (PID=82) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:51:18.928+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:51:18.933+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:51:18.932+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:51:18.964+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:51:19.007+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:51:19.007+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T10:51:19.057+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:51:19.057+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T10:51:19.084+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.172 seconds
[2024-09-20T10:51:49.435+0000] {processor.py:186} INFO - Started process (PID=89) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:51:49.437+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:51:49.441+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:51:49.440+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:51:49.469+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:51:49.572+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:51:49.572+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T10:51:49.625+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:51:49.625+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T10:51:49.663+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.239 seconds
[2024-09-20T10:52:19.837+0000] {processor.py:186} INFO - Started process (PID=96) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:52:19.838+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:52:19.843+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:52:19.842+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:52:19.902+0000] {processor.py:925} INFO - DAG(s) 'new_dag' retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:52:19.975+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:52:19.974+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-09-20T10:52:20.022+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:52:20.022+0000] {dag.py:4156} INFO - Setting next_dagrun for new_dag to 2023-01-01 00:00:00+00:00, run_after=2023-01-02 00:00:00+00:00
[2024-09-20T10:52:20.060+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.238 seconds
[2024-09-20T10:52:43.287+0000] {processor.py:186} INFO - Started process (PID=97) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:52:43.289+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:52:43.297+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:52:43.295+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:52:43.392+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:52:43.377+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 25, in <module>
    task1 = BashOperator(
            ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-09-20T10:52:43.400+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:52:43.428+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.160 seconds
[2024-09-20T10:53:13.665+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:53:13.666+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:53:13.670+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:53:13.669+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:53:13.709+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:53:13.696+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 25, in <module>
    task1 = BashOperator(
            ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-09-20T10:53:13.718+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:53:13.749+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.092 seconds
[2024-09-20T10:53:44.158+0000] {processor.py:186} INFO - Started process (PID=111) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:53:44.160+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:53:44.162+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:53:44.162+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:53:44.207+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:53:44.197+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 25, in <module>
    task1 = BashOperator(
            ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-09-20T10:53:44.213+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:53:44.237+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.086 seconds
[2024-09-20T10:54:14.524+0000] {processor.py:186} INFO - Started process (PID=118) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:54:14.525+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:54:14.527+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:54:14.527+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:54:14.584+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:54:14.572+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 25, in <module>
    task1 = BashOperator(
            ^^^^^^^^^^^^
NameError: name 'BashOperator' is not defined
[2024-09-20T10:54:14.594+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:54:14.625+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.115 seconds
[2024-09-20T10:54:35.919+0000] {processor.py:186} INFO - Started process (PID=125) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:54:35.921+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:54:35.924+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:54:35.923+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:54:36.013+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:54:36.005+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:54:36.024+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:54:36.080+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.176 seconds
[2024-09-20T10:55:06.266+0000] {processor.py:186} INFO - Started process (PID=132) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:55:06.267+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:55:06.273+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:55:06.272+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:55:06.331+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:55:06.323+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:55:06.335+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:55:06.363+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.105 seconds
[2024-09-20T10:55:36.627+0000] {processor.py:186} INFO - Started process (PID=139) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:55:36.628+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:55:36.630+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:55:36.630+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:55:36.679+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:55:36.671+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:55:36.682+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:55:36.708+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.089 seconds
[2024-09-20T10:56:07.131+0000] {processor.py:186} INFO - Started process (PID=146) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:56:07.133+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:56:07.135+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:56:07.135+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:56:07.189+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:56:07.177+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:56:07.193+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:56:07.223+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.100 seconds
[2024-09-20T10:56:37.502+0000] {processor.py:186} INFO - Started process (PID=153) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:56:37.503+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:56:37.505+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:56:37.505+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:56:37.552+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:56:37.544+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:56:37.556+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:56:37.582+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.089 seconds
[2024-09-20T10:57:07.983+0000] {processor.py:186} INFO - Started process (PID=160) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:57:07.985+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:57:07.987+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:57:07.987+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:57:08.024+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:57:08.019+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:57:08.026+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:57:08.051+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.077 seconds
[2024-09-20T10:57:38.378+0000] {processor.py:186} INFO - Started process (PID=167) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:57:38.380+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:57:38.382+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:57:38.381+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:57:38.428+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:57:38.418+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:57:38.431+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:57:38.452+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.084 seconds
[2024-09-20T10:58:08.707+0000] {processor.py:186} INFO - Started process (PID=174) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:58:08.709+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:58:08.710+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:58:08.710+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:58:08.743+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:58:08.739+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:58:08.745+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:58:08.805+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.106 seconds
[2024-09-20T10:58:39.265+0000] {processor.py:186} INFO - Started process (PID=181) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:58:39.267+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:58:39.269+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:58:39.268+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:58:39.311+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:58:39.304+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:58:39.315+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:58:39.379+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.125 seconds
[2024-09-20T10:59:09.647+0000] {processor.py:186} INFO - Started process (PID=188) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:59:09.650+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:59:09.653+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:59:09.652+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:59:09.692+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:59:09.687+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:59:09.695+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:59:09.729+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.094 seconds
[2024-09-20T10:59:40.119+0000] {processor.py:186} INFO - Started process (PID=195) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:59:40.120+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:59:40.123+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:59:40.122+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:59:40.153+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:59:40.149+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    from my_script import my_python_function
ModuleNotFoundError: No module named 'my_script'
[2024-09-20T10:59:40.155+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:59:40.217+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.111 seconds
[2024-09-20T10:59:56.321+0000] {processor.py:186} INFO - Started process (PID=202) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T10:59:56.323+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T10:59:56.342+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:59:56.342+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T10:59:56.387+0000] {logging_mixin.py:190} INFO - [2024-09-20T10:59:56.377+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T10:59:56.393+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T10:59:56.416+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.106 seconds
[2024-09-20T11:00:26.830+0000] {processor.py:186} INFO - Started process (PID=209) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:00:26.832+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:00:26.834+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:00:26.833+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:00:26.893+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:00:26.871+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:00:26.901+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:00:26.939+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.120 seconds
[2024-09-20T11:00:57.260+0000] {processor.py:186} INFO - Started process (PID=216) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:00:57.261+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:00:57.264+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:00:57.263+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:00:57.314+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:00:57.302+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:00:57.323+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:00:57.384+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.134 seconds
[2024-09-20T11:01:27.712+0000] {processor.py:186} INFO - Started process (PID=223) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:01:27.713+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:01:27.716+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:01:27.715+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:01:27.750+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:01:27.737+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:01:27.760+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:01:27.819+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.117 seconds
[2024-09-20T11:01:58.098+0000] {processor.py:186} INFO - Started process (PID=230) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:01:58.100+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:01:58.102+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:01:58.101+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:01:58.143+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:01:58.131+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:01:58.151+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:01:58.180+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.090 seconds
[2024-09-20T11:02:28.581+0000] {processor.py:186} INFO - Started process (PID=237) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:02:28.582+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:02:28.584+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:02:28.584+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:02:28.625+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:02:28.610+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:02:28.634+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:02:28.700+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.130 seconds
[2024-09-20T11:02:59.154+0000] {processor.py:186} INFO - Started process (PID=244) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:02:59.163+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:02:59.166+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:02:59.166+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:02:59.203+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:02:59.192+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:02:59.210+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:02:59.234+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.089 seconds
[2024-09-20T11:03:29.740+0000] {processor.py:186} INFO - Started process (PID=251) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:03:29.743+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:03:29.746+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:03:29.745+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:03:29.797+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:03:29.784+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:03:29.806+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:03:29.840+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.109 seconds
[2024-09-20T11:04:00.109+0000] {processor.py:186} INFO - Started process (PID=258) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:04:00.111+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:04:00.113+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:04:00.112+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:04:00.145+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:04:00.133+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:04:00.158+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:04:00.188+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.088 seconds
[2024-09-20T11:04:30.563+0000] {processor.py:186} INFO - Started process (PID=265) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:04:30.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:04:30.568+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:04:30.567+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:04:30.617+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:04:30.605+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:04:30.625+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:04:30.688+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.132 seconds
[2024-09-20T11:05:00.937+0000] {processor.py:186} INFO - Started process (PID=271) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:05:00.939+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:05:00.944+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:05:00.943+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:05:00.998+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:05:00.982+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:05:01.013+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:05:01.044+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.115 seconds
[2024-09-20T11:05:31.415+0000] {processor.py:186} INFO - Started process (PID=278) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:05:31.417+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:05:31.419+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:05:31.418+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:05:31.466+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:05:31.454+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:05:31.474+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:05:31.528+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.122 seconds
[2024-09-20T11:06:01.788+0000] {processor.py:186} INFO - Started process (PID=285) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:06:01.789+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:06:01.791+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:06:01.791+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:06:01.821+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:06:01.810+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:06:01.830+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:06:01.894+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.115 seconds
[2024-09-20T11:06:32.256+0000] {processor.py:186} INFO - Started process (PID=292) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:06:32.257+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:06:32.260+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:06:32.259+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:06:32.290+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:06:32.280+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:06:32.296+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:06:32.320+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.070 seconds
[2024-09-20T11:07:02.611+0000] {processor.py:186} INFO - Started process (PID=299) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:07:02.612+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:07:02.614+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:07:02.614+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:07:02.652+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:07:02.641+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:07:02.659+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:07:02.719+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.118 seconds
[2024-09-20T11:07:33.045+0000] {processor.py:186} INFO - Started process (PID=307) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:07:33.046+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:07:33.048+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:07:33.048+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:07:33.088+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:07:33.074+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:07:33.097+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:07:33.165+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.136 seconds
[2024-09-20T11:08:03.426+0000] {processor.py:186} INFO - Started process (PID=314) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:08:03.427+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:08:03.431+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:08:03.430+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:08:03.474+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:08:03.460+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:08:03.484+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:08:03.548+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.131 seconds
[2024-09-20T11:08:33.975+0000] {processor.py:186} INFO - Started process (PID=321) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:08:33.976+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:08:33.978+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:08:33.978+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:08:34.033+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:08:34.019+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:08:34.041+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:08:34.109+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.142 seconds
[2024-09-20T11:09:04.379+0000] {processor.py:186} INFO - Started process (PID=328) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:09:04.381+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:09:04.386+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:09:04.384+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:09:04.442+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:09:04.428+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:09:04.451+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:09:04.517+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.145 seconds
[2024-09-20T11:09:34.838+0000] {processor.py:186} INFO - Started process (PID=334) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:09:34.839+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:09:34.842+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:09:34.841+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:09:34.901+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:09:34.886+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:09:34.910+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:09:34.938+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.108 seconds
[2024-09-20T11:10:05.235+0000] {processor.py:186} INFO - Started process (PID=341) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:10:05.237+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:10:05.239+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:10:05.239+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:10:05.283+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:10:05.270+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:10:05.292+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:10:05.355+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.127 seconds
[2024-09-20T11:10:35.653+0000] {processor.py:186} INFO - Started process (PID=348) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:10:35.654+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:10:35.656+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:10:35.656+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:10:35.705+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:10:35.691+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:10:35.712+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:10:35.757+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.114 seconds
[2024-09-20T11:11:06.191+0000] {processor.py:186} INFO - Started process (PID=355) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:11:06.193+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:11:06.196+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:11:06.195+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:11:06.233+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:11:06.222+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:11:06.240+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:11:06.269+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.090 seconds
[2024-09-20T11:11:36.618+0000] {processor.py:186} INFO - Started process (PID=362) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:11:36.620+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:11:36.625+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:11:36.624+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:11:36.662+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:11:36.644+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:11:36.670+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:11:36.701+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.095 seconds
[2024-09-20T11:12:07.109+0000] {processor.py:186} INFO - Started process (PID=369) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:12:07.111+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:12:07.113+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:12:07.113+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:12:07.144+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:12:07.133+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:12:07.152+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:12:07.218+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.117 seconds
[2024-09-20T11:12:37.599+0000] {processor.py:186} INFO - Started process (PID=377) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:12:37.600+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:12:37.602+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:12:37.602+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:12:37.634+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:12:37.625+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:12:37.641+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:12:37.704+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.112 seconds
[2024-09-20T11:13:07.968+0000] {processor.py:186} INFO - Started process (PID=384) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:13:07.971+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:13:07.974+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:13:07.973+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:13:08.034+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:13:08.018+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:13:08.043+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:13:08.110+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.151 seconds
[2024-09-20T11:13:38.501+0000] {processor.py:186} INFO - Started process (PID=391) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:13:38.502+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:13:38.505+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:13:38.505+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:13:38.549+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:13:38.535+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:13:38.558+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:13:38.582+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.088 seconds
[2024-09-20T11:14:08.879+0000] {processor.py:186} INFO - Started process (PID=398) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:14:08.880+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:14:08.882+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:14:08.882+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:14:08.916+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:14:08.906+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:14:08.922+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:14:08.984+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.115 seconds
[2024-09-20T11:14:39.329+0000] {processor.py:186} INFO - Started process (PID=405) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:14:39.330+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:14:39.332+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:14:39.332+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:14:39.382+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:14:39.368+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:14:39.392+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:14:39.455+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.136 seconds
[2024-09-20T11:15:09.722+0000] {processor.py:186} INFO - Started process (PID=412) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:15:09.724+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:15:09.726+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:15:09.726+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:15:09.760+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:15:09.752+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:15:09.767+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:15:09.829+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.115 seconds
[2024-09-20T11:15:40.277+0000] {processor.py:186} INFO - Started process (PID=419) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:15:40.281+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:15:40.289+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:15:40.287+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:15:40.360+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:15:40.341+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:15:40.375+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:15:40.430+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.166 seconds
[2024-09-20T11:16:10.672+0000] {processor.py:186} INFO - Started process (PID=426) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:16:10.674+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:16:10.681+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:16:10.680+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:16:10.724+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:16:10.710+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:16:10.731+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:16:10.761+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.102 seconds
[2024-09-20T11:16:41.040+0000] {processor.py:186} INFO - Started process (PID=433) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:16:41.043+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:16:41.045+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:16:41.045+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:16:41.089+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:16:41.076+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:16:41.097+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:16:41.123+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.098 seconds
[2024-09-20T11:17:11.464+0000] {processor.py:186} INFO - Started process (PID=440) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:17:11.465+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:17:11.468+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:17:11.467+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:17:11.508+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:17:11.495+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:17:11.515+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:17:11.595+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.139 seconds
[2024-09-20T11:17:41.861+0000] {processor.py:186} INFO - Started process (PID=447) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:17:41.863+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:17:41.865+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:17:41.865+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:17:41.910+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:17:41.897+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:17:41.919+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:17:41.951+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.099 seconds
[2024-09-20T11:18:12.406+0000] {processor.py:186} INFO - Started process (PID=454) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:18:12.408+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:18:12.409+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:18:12.409+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:18:12.456+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:18:12.443+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:18:12.464+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:18:12.494+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.099 seconds
[2024-09-20T11:18:42.967+0000] {processor.py:186} INFO - Started process (PID=461) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:18:42.968+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:18:42.970+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:18:42.969+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:18:43.187+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:18:42.997+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:18:43.201+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:18:43.234+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.278 seconds
[2024-09-20T11:19:13.419+0000] {processor.py:186} INFO - Started process (PID=468) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:19:13.421+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:19:13.423+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:19:13.423+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:19:13.470+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:19:13.458+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:19:13.479+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:19:13.509+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.098 seconds
[2024-09-20T11:19:43.772+0000] {processor.py:186} INFO - Started process (PID=475) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:19:43.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:19:43.776+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:19:43.776+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:19:43.813+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:19:43.801+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6, in <module>
    dag = DAG('tutorial', default_args=default_args)
                                       ^^^^^^^^^^^^
NameError: name 'default_args' is not defined
[2024-09-20T11:19:43.822+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:19:43.875+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.113 seconds
[2024-09-20T11:20:04.196+0000] {processor.py:186} INFO - Started process (PID=482) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:04.198+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:20:04.204+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:04.199+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:04.228+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:04.226+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 4
    @task
SyntaxError: invalid syntax
[2024-09-20T11:20:04.229+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:04.255+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.067 seconds
[2024-09-20T11:20:10.276+0000] {processor.py:186} INFO - Started process (PID=483) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:10.277+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:20:10.279+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:10.279+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:10.316+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:10.314+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 4
    @task(task_id= '')
SyntaxError: invalid syntax
[2024-09-20T11:20:10.317+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:10.355+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.088 seconds
[2024-09-20T11:20:17.437+0000] {processor.py:186} INFO - Started process (PID=484) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:17.438+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:20:17.440+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:17.440+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:17.503+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:17.501+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 4
    @task(task_id= 'my_dag')
SyntaxError: invalid syntax
[2024-09-20T11:20:17.507+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:17.546+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.124 seconds
[2024-09-20T11:20:24.661+0000] {processor.py:186} INFO - Started process (PID=491) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:24.663+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:20:24.666+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:24.665+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:24.706+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:24.704+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 6
    
    ^
IndentationError: expected an indented block after function definition on line 5
[2024-09-20T11:20:24.707+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:24.775+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.129 seconds
[2024-09-20T11:20:40.909+0000] {processor.py:186} INFO - Started process (PID=492) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:40.911+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:20:40.913+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:40.912+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:40.941+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:20:40.936+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 4, in <module>
    @task(task_id= 'my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:20:40.943+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:20:41.011+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.111 seconds
[2024-09-20T11:21:04.289+0000] {processor.py:186} INFO - Started process (PID=500) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:21:04.291+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:21:04.293+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:21:04.292+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:21:04.377+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:21:04.371+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:21:04.381+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:21:04.414+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.141 seconds
[2024-09-20T11:21:34.644+0000] {processor.py:186} INFO - Started process (PID=507) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:21:34.646+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:21:34.648+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:21:34.648+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:21:34.679+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:21:34.670+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:21:34.690+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:21:34.769+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.134 seconds
[2024-09-20T11:22:05.154+0000] {processor.py:186} INFO - Started process (PID=514) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:22:05.156+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:22:05.158+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:22:05.158+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:22:05.194+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:22:05.187+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:22:05.198+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:22:05.259+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.117 seconds
[2024-09-20T11:22:35.498+0000] {processor.py:186} INFO - Started process (PID=522) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:22:35.500+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:22:35.504+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:22:35.503+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:22:35.533+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:22:35.523+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:22:35.537+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:22:35.604+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.117 seconds
[2024-09-20T11:23:05.870+0000] {processor.py:186} INFO - Started process (PID=530) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:23:05.871+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:23:05.873+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:23:05.873+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:23:05.895+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:23:05.888+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:23:05.899+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:23:05.932+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.071 seconds
[2024-09-20T11:23:36.383+0000] {processor.py:186} INFO - Started process (PID=537) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:23:36.384+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:23:36.386+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:23:36.386+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:23:36.423+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:23:36.414+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:23:36.426+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:23:36.452+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.077 seconds
[2024-09-20T11:24:06.736+0000] {processor.py:186} INFO - Started process (PID=544) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:24:06.737+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:24:06.739+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:24:06.739+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:24:06.760+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:24:06.754+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:24:06.762+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:24:06.822+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.096 seconds
[2024-09-20T11:24:37.109+0000] {processor.py:186} INFO - Started process (PID=551) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:24:37.112+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:24:37.115+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:24:37.114+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:24:37.146+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:24:37.139+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:24:37.148+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:24:37.220+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.133 seconds
[2024-09-20T11:25:07.527+0000] {processor.py:186} INFO - Started process (PID=558) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:25:07.530+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:25:07.532+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:25:07.531+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:25:07.563+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:25:07.556+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:25:07.565+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:25:07.626+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.109 seconds
[2024-09-20T11:25:37.900+0000] {processor.py:186} INFO - Started process (PID=565) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:25:37.901+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:25:37.903+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:25:37.903+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:25:37.936+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:25:37.931+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:25:37.939+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:25:38.000+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.110 seconds
[2024-09-20T11:26:08.315+0000] {processor.py:186} INFO - Started process (PID=572) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:26:08.317+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:26:08.318+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:26:08.318+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:26:08.338+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:26:08.334+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:26:08.343+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:26:08.404+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.103 seconds
[2024-09-20T11:26:38.634+0000] {processor.py:186} INFO - Started process (PID=579) to work on /opt/airflow/dags/new_dag.py
[2024-09-20T11:26:38.635+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/new_dag.py for tasks to queue
[2024-09-20T11:26:38.638+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:26:38.637+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/new_dag.py
[2024-09-20T11:26:38.660+0000] {logging_mixin.py:190} INFO - [2024-09-20T11:26:38.654+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/new_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/new_dag.py", line 5, in <module>
    @task(task_id='my_dag')
     ^^^^
NameError: name 'task' is not defined
[2024-09-20T11:26:38.663+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/new_dag.py
[2024-09-20T11:26:38.690+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/new_dag.py took 0.064 seconds
